src/tabs/presets/presets.js

```diff
@@ -465,11 +477,23 @@ presets.getFitPresets = function(searchParams) {
         }
     }
 
-    result.sort((a, b) => (a.priority > b.priority) ? -1 : 1);
+    result.sort((a, b) => this.presetSearchPriorityComparer(a,b));
 
     return result;
 };
 
+presets.presetSearchPriorityComparer = function(presetA, presetB) {
+    if (presetA.lastPickDate && presetB.lastPickDate) {
+        return (presetA.lastPickDate > presetB.lastPickDate) ? -1 : 1;
+    }
```

Comment:
If it's a number you also can do (can't remember the order, but pattern holds)
```suggestion
    if (presetA.lastPickDate && presetB.lastPickDate) {
        return presetB.lastPickDate - presetA.lastPickDate
    }
```

Reply:
its a date-time, i will give it a try...